### PR TITLE
s20 update

### PR DESCRIPTION
**Pull request by brandonleung**
- changed position descriptions
- split VPX to sponsorships and scoping 
- added sponsorship-specific questions
- opening apps for June 22

### CODE REVIEW

File: e2e/utility.ts [hunk 1]
```diff
@@ -106,7 +106,7 @@ export function selectAndFillProjectLeadQuestions(positionNumber: number, roleQu
 
 export function selectVPExternal(positionNumber: number) {
```
Comment (sherryhli) [inline e2e/utility.ts:107]: Let's rename this to be `selectVPExternalSponsorship`

It'll be great if we can write E2E tests for the new position as well but that can be done in a separate PR if we want to get applications opened quickly.

Comment (brandonleung) [reply e2e/utility.ts]: Yep, will rename now and can add different E2E tests later on

File: e2e/utility.ts [hunk 2]
```diff
@@ -104,14 +104,14 @@ export function selectAndFillProjectLeadQuestions(positionNumber: number, roleQu
 }
 
 
-export function selectVPExternal(positionNumber: number) {
+export function selectVPExternalSponsorship(positionNumber: number) {
```
Comment (MickeyDang) [inline e2e/utility.ts:107]: Is there supposed to be a similar function for `selectVPExternalScoping()` ?

Comment (brandonleung) [reply e2e/utility.ts:107]: yes but no lol .. will get around to this later. i tested it manually for now

**Pull request merged**